extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -453,6 +471,23 @@ export class AmpStory360 extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Ensures user is facing a specified point of interest.
+   * @private
+   */
+  maybeSetGyroscopeDefaultHeading_() {
+    if (this.isOnActivePage_ && this.gyroscopeControls_ && this.isReady_) {
+      this.headingOffset_ =
+        parseFloat(
+          this.element.getAttribute('heading-end') ||
+            this.element.getAttribute('heading-start') ||
+            0
+        ) +
+        90 +
```

Comment:
maybe comment that magic number

Reply:
We can remove it here if we also remove it in the heading calculation in `CameraOrientation.fromDegrees`:
https://github.com/ampproject/amphtml/blob/6120560b8bd2d449870fbd9b35684a7cf3fee0b0/extensions/amp-story-360/0.1/amp-story-360.js#L141
Wdyt?

Reply:
Ah! OK now I get it, that's because of the different conventions around the center value. As currently implemented the value zero is the center [-90; 90] to match some convention, probably the street view api one given the origins of the extension, instead of [0;180]. We can't change that behaviour now, that would break the existing stories.

Reply:
Awesome. Thanks for explaining this. 
I revised this and created a top level const called `CENTER_OFFSET`.